### PR TITLE
bug 1227142 - crontabber app to clean up raw_adi

### PR DESCRIPTION
**Pull request by peterbe**

### CODE REVIEW

**Comment by peterbe** [comment]
Actually @twobraids I don't think I want a "single transaction". I don't want to use transactions at all. Do we have a decorator for that?

File: socorro/cron/jobs/clean_raw_adi.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+from configman import Namespace
+from crontabber.base import BaseCronApp
+from crontabber.mixins import (
+    with_postgres_transactions,
+    with_single_postgres_transaction,
+)
+
+
+@with_postgres_transactions()
+@with_single_postgres_transaction()
```
Comment (adngdb) [inline socorro/cron/jobs/clean_raw_adi.py]: Did you figure out that thing about having no transaction at all?

**Comment by adngdb** [comment]
This is essentially the same as your other PR, so r+.

File: socorro/cron/jobs/clean_raw_adi.py (hunk 1 above)
Comment (peterbe) [inline socorro/cron/jobs/clean_raw_adi.py]: Nope and I'm not interested. I manually deleted all the old cruft. 
This new job is going to run every day so the amount of stuff it deletes is only going to be a days worth.

**Pull request merged**